src/lib/perceived-latency-instrumentation.js

```diff
@@ -0,0 +1,96 @@
+/* @flow */
+import { getLogger } from '@paypal/sdk-client/src';
+import { FPTI_KEY } from '@paypal/sdk-constants/src';
+
+/* To Track time spent in each phase(cdn download, chunks download, etc)
+    logLatencyInstrumentationPhase('first_interactable_render')
+    logLatencyInstrumentationPhase('html_body', 'comp', true);
+    logLatencyInstrumentationPhase('html_body', 'comp');
+*/
+const latencyInstrumentationPhases = {};
+export const logLatencyInstrumentationPhase = (buttonID : string, phase : string, category : string, isStart : ?boolean) => {
```

Comment:
Could you convert this function to take in a single object instead of 4 arguments? That will help it to scale better it the future and also makes the caller code more readable. Ex:

```js
logLatencyInstrumentationPhase(props.buttonSessionID, 'first_render', 'comp', true);
```

vs:

```js
logLatencyInstrumentationPhase({ 
  buttonID: props.buttonSessionID, 
  phase: 'first_render', 
  category: 'comp',  
  isStart: true 
});
```